access/views.py

```diff
@@ -39,10 +46,125 @@ def index(request):
         })
     return render(request, 'access/ready.html', {
         "courses": courses,
-        "manager": 'gitmanager' in settings.INSTALLED_APPS,
     })
 
 
+def publish(request):
```

Comment:
Is `login_required` missing on purpose?

Reply:
Yes, this is called from configure which has login_required instead of being directly accessible.